src/Core/Addon/Module/ModuleRepositoryInterface.php

```diff
@@ -63,4 +63,13 @@ public function getList();
      * @return ModuleInterface
      */
     public function getModule($name);
+
+    /**
+     * Get Module Attributes (name, displayName, etc.).
+     *
+     * @param string $name The technical name of the module
+     *
+     * @return \Symfony\Component\HttpFoundation\ParameterBag
+     */
+    public function getModuleAttributes($name);
```

Comment:
```suggestion
    public function getModuleAttributes(string $name): ParameterBag;
```

Reply:
Same question as for isActive, should i update the class \PrestaShop\PrestaShop\Core\Addon\Module\ModuleRepository too ?